andela/ah-code-blooded
# #161982021 Fix article sharing links on the article payload

**What does this PR do?**

This PR fixes the article sharing links available on the article payload. The current behavior is that the links provide point back to the backend while they're actually supposed to point to the client.

**Description of Task to be completed?**

##### Steps to reproduce
1. Load an article from the backend.
2. Inspect the `share_article` data.
3. Examine the share links provided.

##### Expected
The share links provided should point to the frontend application.

##### Actual
The share links provided point to the backend application.

**How should this be manually tested?**

This can be manually tested locally by following these steps:
1. Ensure that the `CLIENT_DOMAIN` env variable is set.
2. Set the `CLIENT_READ_ARTICLE_ROUTE` env variable.
3. Export env variables: `source .env`
3. Run the application: `python manage.py runserver`
4. Using Postman create an article.
5. Examine the response. It should contain a `share_article` property that contains links to share the article via Email, Twitter, Facebook and LinkedIn. The links should indicate that the article points to the client app.

NB. Examples of the two env variables are included in the `.env_example` file.

**What are the relevant pivotal tracker stories?**

[#161982021](https://www.pivotaltracker.com/story/show/161982021)

**Screenshots**
In this example `CLIENT_DOMAIN` is set to `http://localhost:3000` while `CLIENT_READ_ARTICLE_ROUTE` is set to `articles`.

<img width="972" alt="screen shot 2018-11-15 at 18 12 40" src="https://user-images.githubusercontent.com/8180548/48561933-24b75080-e902-11e8-9a8d-7ba1a5fe26ad.png">
